cmd/nodepool/create.go

```diff
@@ -51,6 +58,10 @@ func NewCreateCommand() *cobra.Command {
 		RootVolumeType: "gp2",
 		RootVolumeSize: 120,
 		RootVolumeIOPS: 0,
+		// TODO nargaman need to move this into platform specific create file
```

Comment:
nit: usually we use this convention ordered by preference:
// TODO (jira-1234): desc
// TODO (username): desc
// TODO: desc

Reply:
Changed my TODO comments to this format (`// TODO (username): desc`)